### DISCUSSION TITLE
Is it possible to change the direction of the soft shadows when using AccumulativeShadows ?

### DISCUSSION THREAD

**Discussion by ymolists** [pmndrs/drei#1191] [Q&A]
I am trying to render the soft shadow to be on the right as if the lights were coming from the left instead of behind the scene.

![image](https://user-images.githubusercontent.com/6591993/208429270-dc3cc625-891d-49ac-86a3-18d2f500a6f3.png)

**Comment by drcmda** [comment]
yes. on the randomized light there is a position prop.
